### ISSUE TITLE
Incompatibility with Dash-to-Panel Scroll Action

### ISSUE THREAD

**Issue by Batwam** [Moon-0xff/gnome-mpris-label#47]
Hello,

I just noticed as I was testing with multiple workspaces that the scroll action on the label was also trigerring the workspaces to switch. When I turned of Dash-to-Panel, this went away so this is not an issue without the extension. I then looked into the settings and noticed that by default, scroll will `Switch workspace`
![image](https://user-images.githubusercontent.com/56710655/229516781-e8980c9d-8419-40e2-807d-903f16e21e9c.png)

I checked and we do have `return Clutter.EVENT_STOP;` in `_onScroll(event)` but clearly both extensions are getting the signal. 

This issue can be disabled which is what I've done but I just wanted to put this out there so we are aware of this limitation.

**Comment by Moon-0xff** [comment]
Should we close this one? Both extensions have options to disable the scroll actions, ours since 34829624994af461f4b1f773ba78160488606307.

We could try to prevent the signal from reaching both extensions (somehow). But ultimately I don't think we should implement complicated stuff for the weird edge cases.

**Comment by Batwam** [comment]
Yeah, it's a known issue but I don't think that there is much we can do about. We could perhaps include a know in the Readme to state that it's a know limitation?
